composer.json

```diff
@@ -4,31 +4,35 @@
     "description": "CrefoPay module",
     "license": "MIT",
     "require": {
-        "php": ">=7.3",
+        "php": ">=7.4",
         "spryker-eco/crefo-pay-api": "^1.0.0",
         "spryker-shop/checkout-page": "^1.5.0 || ^2.0.0 || ^3.0.0",
         "spryker/calculation": "^4.0.0",
         "spryker/checkout-extension": "^1.0.0",
+        "spryker/decimal-object": "^1.0.0",
         "spryker/kernel": "^3.30.0",
         "spryker/locale": "^3.0.0",
         "spryker/oms": "^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
         "spryker/propel-orm": "^1.6.0",
         "spryker/refund": "^5.0.0",
-        "spryker/sales": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
-        "spryker/step-engine": "^2.0.0 || ^3.0.0",
+        "spryker/sales": "^8.5.0 || ^10.0.0 || ^11.0.0",
+        "spryker/step-engine": "^3.1.0",
         "spryker/symfony": "^3.0.0",
         "spryker/util-text": "^1.0.0",
         "spryker/zed-request": "^3.0.0"
     },
     "require-dev": {
+        "codeception/module-asserts": "^1.3.0",
+        "phpstan/phpstan": "^0.12",
```

Comment:
This must not be that outdated for many months now
We use ^1.2+ is there any reason to use this old one?